package/MDAnalysis/topology/GROParser.pyx

```diff
@@ -47,6 +47,7 @@
 from __future__ import absolute_import
 
 import numpy as np
+cimport numpy as np
```

Comment:
this is only needed when you have numpy on the left hand side of things, eg `cdef np.ndarray[float] arr`.  You're using memoryviews here (which is better), so you just need the Python import to get numpy functions imported.

Reply:
Yes, my bad. Forgot to remove after I shifted to memoryviews.